ui/components/requirements/requirement.jsx

```diff
@@ -7,6 +7,7 @@ export default function Requirement({ requirement }) {
     <span key={idx} className="req-text-line mb1">{ line }<br /></span>
   ));
   /* eslint-enable react/no-array-index-key */
+  const sunsetString = requirement.policy.sunset ? 'Sunset date by $requirement.policy.sunset' : 'Sunset date: none';
```

Comment:
Hmm, I thought that was the right way to do string interpolation in this context, will investigate further.

Tests are good, I will add some.